### ISSUE TITLE
Fetching completed workflow ID 

### ISSUE THREAD

**Issue by ibrahimalafeef2** [firdaus/cadence-python#20]
Is there any way to fetch the completed activity/workflow in the logger?

The DEBUG logger is returning the status of the workflow, created and running is returning with the workflow ID, but the completed workflow does not return the workflow ID. 

I am using async in the client to hit the request, and fetching the statuses from the logs whether the workflow is created, running, completed or failed. 

Thanks

**Comment by firdaus** [comment]
Yeah I think that's doable, I can place a log message with the workflow id when the workflow method returns.
